src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/events/DeleteTakeEvent.kt

```diff
@@ -0,0 +1,15 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.takecard.events
+
+import javafx.event.Event
+import javafx.event.EventType
+import org.wycliffeassociates.otter.common.data.workbook.Take
+
+class DeleteTakeEvent(
+    type: EventType<DeleteTakeEvent>,
```

Comment:
Do you need to accept type in the constructor?  Can you just use the companion value?